blockchain/staking/run_a_validator_non_technical.md

```diff
@@ -0,0 +1,71 @@
+# Run a validator
+
+## Introduction
+
+This guide will walk you through the process of setting up a new validator on Liberland Chain for non technical people.
+[You can follow along a video here](https://www.youtube.com/watch?v=ufV7igODxcQ)
+## Requirements
+
+* Liberland wallet with at least 200 LLD. [Ask to be onboarded](https://matrix.to/#/#liberland-node:matrix.org)
+* [PolkadotJs wallet](https://polkadot.js.org/extension/)
+* A helper [Ask here if you dont have one](https://matrix.to/#/#liberland-node:matrix.org)
+
+## Get wallet
+Go to [PolkadotJs wallet](https://polkadot.js.org/extension/)
+Download extension
+Use the extension and create a new wallet. Save the mnemonic key somewhere safe. If you lose the mnemonic key, your account is lost.
```

Comment:
These render as a single line. Add `* ` or numbers before each line to make it a list.